components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -109,8 +109,10 @@ func main() {
 	//
 	// Using map is intentional - we ensure that component name is not duplicated.
 	optionalComponentsDisablers := runtime.ComponentsDisablers{
-		"Kiali":  runtime.NewGenericComponentDisabler("kiali", "kyma-system"),
-		"Jaeger": runtime.NewGenericComponentDisabler("jaeger", "kyma-system"),
+		"Kiali":                   runtime.NewGenericComponentDisabler("kiali", "kyma-system"),
+		"Jaeger":                  runtime.NewGenericComponentDisabler("jaeger", "kyma-system"),
+		"KnativeProvisionerNatss": runtime.NewGenericComponentDisabler("knative-provisioner-natss", "knative-eventing"),
+		"NatssStreaming":          runtime.NewGenericComponentDisabler("nats-streaming", "natss"),
```

Comment:
Please add explicit comment under those two components that:
> TODO(workaround): those components should be always disabled and user should not be able to enable them in provisioning request. This mean those component cannot be specified under the plan schema definition.

Reply:
Added